mozetl/taar/taar_lite_guidguid.py

```diff
@@ -0,0 +1,189 @@
+"""
+This ETL job computes the co-installation occurrence of white-listed
+Firefox webextensions for a sample of the longitudinal telemetry dataset.
+"""
+
+import click            # noqa
+import boto3            # noqa
+import datetime as dt   # noqa
+import json             # noqa
+import logging          # noqa
+from botocore.exceptions import ClientError   # noqa
+from pyspark.sql import Row, SparkSession     # noqa
+from pyspark.sql.functions import col, collect_list, explode, udf, sum as sum_, max as max_, first  # noqa
```

Comment:
I usually get around the namespacing conflicts by using `import pyspark.sql.functions as F`. The `sum_` and `max_` could benefit from namespacing.

Reply:
This seems like an arbitrary style decision that won't help with the clarity of the code.  I'd rather leave the trailing underscore notation because it's more concise.

Reply:
Hey, while I also like the trailing underscores, Dexterp37 has pointed out that the `import as F` is conventional for this repo. Let's refactor for consistency within the python_mozetl standards.

Sorry for the confusion, this was my code.